troupon/static/scss/partials/_resets.scss

```diff
@@ -49,8 +49,14 @@ ul, ol {
     position: absolute;
     z-index: z-index(dropdown);
 
-    li > a {
+    & > li {
+      width: 100%;
+      padding: 0;
+    }
+
+    & > li > a {

```

Comment:
Merge rule `& > li > a` with rule on line 52<br>Selector should have depth of applicability no greater than 2, but was 4<br>Unnecessary parent selector (&)